Fabric.Identity.API/scripts/Install-Identity-Utilities.psm1

```diff
@@ -0,0 +1,642 @@
+# Import Fabric Install Utilities
+$fabricInstallUtilities = ".\Fabric-Install-Utilities.psm1"
+if (!(Test-Path $fabricInstallUtilities -PathType Leaf)) {
+    Write-DosMessage -Level "Warning" -Message "Could not find fabric install utilities. Manually downloading and installing"
+    Invoke-WebRequest -Uri https://raw.githubusercontent.com/HealthCatalyst/InstallScripts/master/common/Fabric-Install-Utilities.psm1 -Headers @{"Cache-Control" = "no-cache"} -OutFile $fabricInstallUtilities
+}
+Import-Module -Name $fabricInstallUtilities -Force
+
+$dosInstallUtilities = Get-Childitem -Path ./**/DosInstallUtilities.psm1 -Recurse
+if ($dosInstallUtilities.length -eq 0) {
+    Install-Module DosInstallUtilities -Scope CurrentUser
+    Import-Module DosInstallUtilities -Force
```

Comment:
I had been using force for the dbatools that is used by DosInstallUtilities. I had an issue with folders created for each version change, and that led to a 10 level deep folder nesting error. Since force retains all installed version changes in folders, if DosInstallUtilities has a lot of version changes we may need to clean out the folders or not use force. Just a thought.

Reply:
I'll update this so it only installs if we don't have the version we want, I think that should take care of this potential issue.